src/providerProps.js

```diff
@@ -9,6 +9,8 @@ export default {
   renderLoading: PropTypes.oneOfType([PropTypes.func, PropTypes.bool]), // undefined -> Html
   renderError: PropTypes.oneOfType([PropTypes.func, PropTypes.bool]), // error -> Html
   afterSuccess: PropTypes.func, // (response, json)
+  afterError: PropTypes.func, // (response, json)
```

Comment:
The term we use in actions and reducers is `failure`. This should be called `afterFailure`